integrationTests/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "commerce-sdk-integration-test",
+  "version": "1.4.1-beta.0",
+  "description": "Salesforce Commerce Cloud SDK Integration Test",
+  "homepage": "https://developer.commercecloud.com/s/",
+  "scripts": {
+    "depcheck": "depcheck",
+    "lint": "eslint \"**/*.ts\" --quiet --fix",
+    "pretest": "npm run lint && npm run depcheck",
+    "test": "nyc mocha \"tests/*.test.*\"",
+    "fetch:internal": "npm pack ../packages/core && npm pack ../packages/generator && npm install ",
+    "test:integration": "npm run fetch:internal &&  npm run test",
+    "test:integration:redis": "fetch:internal && npm run redis --prefix ../packages/core && npm run test; npm run redis:shutdown --prefix ../packages/core",
+    "test:integration:redis:debug": "fetch:internal && npm run redis --prefix ../packages/core && npm run test; npm run redis:shutdown:debug --prefix ../packages/core"
+  },
+  "dependencies": {
+    "@commerce-apps/core": "file:./commerce-apps-core-1.4.1-beta.0.tgz",
```

Comment:
How do we keep this version in line when we update the version?

Reply:
I'll test if you can use a wildcard, If not, I didn't see a way to re-use the version from the package.json as a variable, but I'll take another look

Reply:
resolved